packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -309,17 +325,8 @@ class MethodChannel {
   ///  * <https://api.flutter.dev/javadoc/io/flutter/plugin/common/MethodCall.html>
   ///    for how to access method call arguments on Android.
   @optionalTypeArgs
-  Future<T> invokeMethod<T>(String method, [ dynamic arguments ]) async {
-    assert(method != null);
-    final ByteData result = await binaryMessenger.send(
-      name,
-      codec.encodeMethodCall(MethodCall(method, arguments)),
-    );
-    if (result == null) {
-      throw MissingPluginException('No implementation found for method $method on channel $name');
-    }
-    final T typedResult = codec.decodeEnvelope(result) as T;
-    return typedResult;
+  Future<T> invokeMethod<T>(String method, [ dynamic arguments ]) {
+    return _invokeMethod(method, missingOk: false, arguments: arguments);
```

Comment:
You may want to pass `T` as well (i.e. `return _invokeMethod<T>(...)`)?

Reply:
Done